### DISCUSSION TITLE
Sound delay on steam deck

### DISCUSSION THREAD

**Discussion by Waldiiiii** [ppy/osu#22448] [Q&A] [answered]
### Type

Game behaviour

### Bug description

I started the game on the steam deck, used the same settings I use on windows and started different Maps to play.
everytime after the first cycle appears there is a small lag and than the clicking sound is delayed to the click and is slowed behind the Music.

I tried other settings:
changing resolution
turning on and off fullscreen
turning on and off hardware acceleration
using different skins
use 2 different headphones

### Version

2023.123.0

### Logs

[database.log](https://github.com/ppy/osu/files/10527719/database.log)
[input.log](https://github.com/ppy/osu/files/10527720/input.log)
[network.log](https://github.com/ppy/osu/files/10527721/network.log)
[performance.log](https://github.com/ppy/osu/files/10527722/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10527723/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10527724/updater.log)

**Comment by Waldiiiii** [comment, marked as answer]
found the issue: dont use bluetooth. at least not more than 2 devices. 

After I plugged in the headphone the hitsound was normal. but then I tested it again with bluetooth but plugged my mouse in and the sound was also normal. after I used both devices with Bluetooth it became weird again.

I was so confused becouse the music is completly in sync with the cicles (I tested it by turning of effects and only using music on several maps) but only the hitsouns where delayed

**Reply by peppy** [reply]
Bluetooth headsets always have induced latency, no matter which technology. I'd recommend against using wireless headphones for osu! under any circumstance.
